docs/scenarios_authoring/Intro.md

```diff
@@ -6,16 +6,16 @@ sidebar_position: 1
 
 ## Intended audience
 
-Nussknacker provides a drag and drop visual authoring tool (Nussknacker Designer) allowing to define decision algorithms – we call them scenarios – without the need to write code. This document is intended for those who will use Nussknacker Designer to configure processing scenarios. Nussknacker is a low-code platform; prior knowledge of SQL, JSON and concepts like variables and data types will help master authoring of the data processing scenarios in Nussknacker. 
+Nussknacker provides a drag and drop visual authoring tool (Nussknacker Designer). This tool allows the user to define decision algorithms – we call them scenarios – without the need to write code. This document is intended for those who will use Nussknacker Designer to configure scenario processing. Nussknacker is a low-code platform; prior knowledge of SQL, JSON and familiarity with concepts like variables and data types will help master data processing with Nussknacker. 
 
 **Please try the [Demo](/quickstart/demo) to quickly understand how to move around Nussknacker Designer, create a simple scenario and see SpEL in action.**
 
 &nbsp;
 ## Nussknacker scenario diagram
 
-Nussknacker scenario represents a decision algorithm in a form of a graph. The decisions are based on data, which can be anything from clicks on a website, bank transactions, to readings from sensors. These data are processed by Nussknacker, according to the scenario (algorithm). The results generated by Nussknacker are also data; this time they communicate the result or decisions.
+Nussknacker scenario represents a decision algorithm in the form of a graph. The decisions are based on data, which can be anything from clicks on a website, to bank transactions, to readings from sensors. This data is processed by Nussknacker, according to the defined scenario(algorithm). Applying the scenario template to the input the data yields some output from Nussknacker. Either result data or information about decisions to be made.
 
-Every scenario starts with a source of data because we have to know what kind of data we want to work with. The rest of the scenario is a sequence (Directed Acyclic Graph or DAG to be more precise) of different nodes:
+Every scenario starts with a source of data. We have to specify what kind of data we want to work with. The rest of the scenario is a sequence (Directed Acyclic Graph or DAG to be more precise) of different nodes:
```

Comment:
These first 2 sentences (or the original first sentence) are not clear. We need to explain here that in the streaming processing mode one specifies Kafka topic from which events are read. In the R-R processing mode "it is on the diagram for aesthetical reason" (surely a better justification is needed). Anyway, current and previous formulation contains unfinished thought.

Reply:
I think maybe the wording is not optimal, but I would restrain from explaining specific modes here. I think the intro should just present an abstract concept of a data source from which nussknacker can take data.